server/src/models/user.model.js

```diff
@@ -0,0 +1,39 @@
+require("dotenv").config();
+const { DataTypes } = require("sequelize");
+const { sequelize } = require("../configs/sequelize");
+
+const User = sequelize.define(process.env.USERS_TABLE, {
+  id: {
+    type: DataTypes.INTEGER,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+  first_name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  last_name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  email: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    unique: true,
+  },
+  password: {
+    type: DataTypes.BLOB("tiny"),
```

Comment:
For hashing and salting, the password isn't stored as a string, as it hashes it into just straight up bytes, so if we stored it as a string it could cause some issues relating to encodings so to save the headache, I just store the bytes directly